docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -77,6 +77,10 @@ You could also pipe a value from one command to another. This example uses stand
 az vm list --query "[?powerState=='VM running'].name" --output tsv | grep my_vm
 ```
 
+## Pass arguments
+
+### Use quotation marks in arguments
```

Comment:
Without the headings, the following text `When you work with Azure CLI commands,` will appear under the wrong section of `## Pass values to another command`.